backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -450,6 +457,38 @@ class Listing extends BaseEntity {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsNumber({}, { groups: [ValidationsGroupsEnum.default] })
   waitlistOpenSpots?: number | null
+
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  ownerCompany?: string | null
+
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  managementCompany?: string | null
+
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  managementWebsite?: string | null
+
+  // In the absence of AMI percentage information at the unit level, amiPercentageMin and
+  // amiPercentageMax will store the AMI percentage range for the listing as a whole.
+  @Column({ type: "text", nullable: true })
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  amiPercentageMin?: string | null
```

Comment:
Should the amiPercentageMin and amiPercentageMax be enums? Or alternatively maybe they should be integers? 

I wonder if [percentages listed in the income table](https://detroitmi.gov/sites/detroitmi.localhost/files/2020-08/2020%20Income%20Limits.pdf) are the only allowed percentages?

Reply:
I'd guess that you're right that the AMI percentages can only be those in that pdf, so maybe an enum would make sense. I'm going to switch this to a number, though, because I'm realizing that this is likely something we'll want to be able to filter on at the DB level.